cluster/gce/gci/configure.sh

```diff
@@ -549,9 +541,28 @@ function install-containerd-ubuntu {
 }
 
 function install-auth-provider-gcp {
-  # Keep in sync with --image-credential-provider-config in ../util.sh
-  local auth_config_file="${KUBE_HOME}/cri_auth_config.yaml"
-  cat >> "${auth_config_file}" << EOF
+  local -r filename="auth-provider-gcp"
+  local -r auth_provider_storage_full_path="${AUTH_PROVIDER_GCP_STORAGE_PATH}/${AUTH_PROVIDER_GCP_VERSION}/${HOST_PLATFORM}_${HOST_ARCH}/${filename}"
```

Comment:
You probably don't want this, as we will no longer use ECP which is build in this repo.

Reply:
I am not sure I fully understand, but I have reverted this change.